build_tools/waf_content.py

```diff
@@ -83,7 +86,7 @@ def compile(task):
             mod = __import__(module)
             # NOTE: We can't use getattr. msg_type could of form "foo.bar"
             msg = eval('mod.' + msg_type)() # Call constructor on message type
-            with open(task.inputs[0].srcpath(task.env), 'rb') as in_f:
+            with open(task.inputs[0].srcpath(), 'rb') as in_f:
```

Comment:
Function doesn't take an env as argument in newer waf anymore, waf handles that automatically now